ambari-logsearch-logfeeder/src/main/java/org/apache/ambari/logfeeder/conf/LogFeederProps.java

```diff
@@ -199,6 +199,16 @@
   @Value("${" + LogFeederConstants.SOLR_URLS + ":}")
   private String solrUrlsStr;
 
+  @LogSearchPropertyDescription(
+    name = LogFeederConstants.CLOUD_STORAGE_MODE,
+    description = "Option to support sending logs to cloud storage. You can choose between supporting only cloud storage, non-cloud storage or both",
+    examples = {"default", "cloud", "hybrid"},
+    defaultValue = "true",
+    sources = {LogFeederConstants.CLOUD_STORAGE_MODE}
+  )
+  @Value("${" + LogFeederConstants.CLOUD_STORAGE_MODE + ":default}")
+  public String cloudStorageMode;
```

Comment:
Can `cloudStorageMode` have a type of `LogFeederMode` instead of `String`

Reply:
it can be, note that in the Condition classes, we will need to gather that from a string anyway